P-manBrown/tascon-frontend
# chore: resolve errors encountered when running Lefthook

### Summary

To resolve errors occurring when running Lefthook, I will modify the settings for ESLint and Prettier.

### Changes

- Modify ESLint settings
  Since Jest will not be used for a while, I uninstalled `eslint-plugin-jest` and removed references to `eslint-plugin-jest` from `.eslintrc.js`.
  This change resolves ESLint errors when running Lefthook.

- Modify Prettier settings
  To resolve errors when running Prettier, I modified the `plugins` section in `.prettierrc.js`.
  This change resolves Prettier errors when running Lefthook.

### Testing

I confirmed that Lefthook's `pre-commit` works correctly with the following steps:

1. Make changes to any file and stage them.
2. Run `yarn lefthook run pre-commit` in the terminal.
3. Confirm that no errors occur.

I confirmed that Lefthook's `pre-push` works correctly with the following steps:

1. Run `yarn lefthook run pre-push` in the terminal.
2. Confirm that no errors occur.

### Related Issues (Optional)

None

### Notes (Optional)

I referred to the following page for the Prettier settings modification:
[Cannot use any prettier plugins with yarn 4 + PnP](https://is.gd/ja0J4b)
